### PR TITLE
Add personal layout: dz60:mpstewart

### PR DESCRIPTION
**Pull request by mpstewart**
This pull request adds my personal layout for the DZ60.

### CODE REVIEW

File: keyboards/dz60/keymaps/mpstewart/keymap.c [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+ * │lshift  │   │   │   │   │   │   │   │end│pdn│ ↓ │rshift│   │  │30      │32 │33 │34 │35 │36 │37 │38 │39 │3a │3b │3d    │3e │
+ * ├────┬───┴┬──┴─┬─┴───┴───┴───┴───┴───┴──┬┴───┼───┴┬────┬┴───┤  ├────┬───┴┬──┴─┬─┴───┴───┴───┴───┴───┴──┬┴───┼───┴┬────┬┴───┤
+ * │ctrl│lalt│lgui│       S P A C E        │rgui│ralt│menu│ctrl│  │40  │41  │43  │46                      │4a  │4b  │4d  │4e  │
+ * └────┴────┴────┴────────────────────────┴────┴────┴────┴────┘  └────┴────┴────┴────────────────────────┴────┴────┴────┴────┘
+ * XXX = RESET (dfu mode)
+*/
+	[_FN] = LAYOUT_60_ansi_split_bs_rshift(
+ //    _0       _1       _2       _3       _4        _5       _6       _7       _8       _9       _A       _B       _C       _D         _E
+/*0_*/ KC_SLEP, KC_F1,   KC_F2  , KC_F3,   KC_F4,    KC_F5,   KC_F6,   KC_F7,   KC_F8,   KC_F9,   KC_F10,  KC_F11,  KC_F12,  KC_INS,    KC_DEL,
+/*1_*/ KC_CAPS,          KC_BDN,  KC_BUP,  KC_NO,    RESET,   KC_NO,   KC_NO,   KC_NO,   KC_PSCR, KC_SLCK, KC_PAUS, KC_UP,   KC_NO,     KC_NO,
```
Comment (fauxpark) [inline keyboards/dz60/keymaps/mpstewart/keymap.c:71]:
```suggestion
/*1_*/ KC_CAPS,          KC_BRID,  KC_BRIU,  KC_NO,    RESET,   KC_NO,   KC_NO,   KC_NO,   KC_PSCR, KC_SLCK, KC_PAUS, KC_UP,   KC_NO,     KC_NO,
```

File: keyboards/dz60/keymaps/mpstewart/keymap.c [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+/*2_*/ KC_NO,            KC_VDN,  KC_VUP,  KC__MUTE, KC_NO,   KC_NO,   KC_NO,   KC_NO,   KC_NO,   KC_NO,   KC_LEFT, KC_RGHT, KC_RETURN,
```
Comment (fauxpark) [inline keyboards/dz60/keymaps/mpstewart/keymap.c:72]:
```suggestion
/*2_*/ KC_NO,            KC_VOLD,  KC_VOLU,  KC_MUTE, KC_NO,   KC_NO,   KC_NO,   KC_NO,   KC_NO,   KC_NO,   KC_LEFT, KC_RGHT, KC_RETURN,
```

File: keyboards/dz60/keymaps/mpstewart/keymap.c [hunk 3]
```diff
@@ -0,0 +1,99 @@
+#include QMK_KEYBOARD_H
+#define C_ESC CTL_T(KC_ESC)
+#define KC_VUP KC__VOLUP
+#define KC_VDN KC__VOLDOWN
+#define KC_BUP KC_BRIU
+#define KC_BDN KC_BRID
```
Comment (fauxpark) [inline keyboards/dz60/keymaps/mpstewart/keymap.c:6]:
```suggestion
```

File: keyboards/dz60/keymaps/mpstewart/keymap.c [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+
+#define RGBLIGHT_SLEEP
```
Comment (fauxpark) [inline keyboards/dz60/keymaps/mpstewart/keymap.c:8]:
```suggestion
```

**Comment by mpstewart** [comment]
@fauxpark,
Thank you for the suggestions. For my own edification, are macro aliases generally frowned upon in this project for new keymaps?

**Comment by fauxpark** [comment]
Those ones were, since there's no point to them -  QMK already has 7-character short aliases for those keycodes, which are there specifically so you can align your keymap for readability.

**Comment by noroadsleft** [comment]
Thanks!

**Pull request merged**